package.lisp

```diff
@@ -1952,7 +1952,18 @@
    ;; images
    #:rgb-image
    #:xpm-parse-file
-   #:*xpm-x11-colors*))
+   #:*xpm-x11-colors*
+   ;; ellipses
+   #:reparameterize-ellipse
+   #:ellipse-point
+   #:ellipse-point*
+   #:ellipse-derivative
+   #:ellipse-cubic-bezier-control-points
+   #:ellipse-cubic-bezier-control-points*
+   #:ellipse-cubic-bezier-points
+   #:ellipse-cubic-bezier-points*
+   #:transformation-angle
+   #:transform-angle))
 
```

Comment:
do you really think these methods should be part of official McCLIM API advertised outside? If so, please provide documentation what would I potentially need ellipse-derivative function for (etc). I think that these methods should be part of climi and not exported at all unless there is indeed a good usecase for normal application writing to know these. Programmer (imho) is only interested about the fact, that there are ellipsis and how he can draw it, not how it is done.

Reply:
nb: we should probably write some developer guideline with a consistent set of rules for that

Reply:
I don't feel strongly about where these live. They can live in climi if you would like. The PDF backend needs reparameterize-ellipse, ellipse-cubic-bezier-points, and transform-angle. transformation-angle is a useful utility that I could see others wanting. drawing-tests want parameterize-ellipse as well. I plan on using some of these functions in my ellipse drawing code to help make a nice UI to the control points. personally, I'm a fan of lots of little packages but that isn't McCLIM style. Happy to move it all to climi if you would like.

Reply:
then please do (there is no need for exporting these symbols, our backends are part of McCLIM internals anyway, so they can directly `import-from` from climi).